1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,243 @@
+|                                  | ScriptBlockLogging          | EnableScriptBlockLogging           | DWORD  | Computer, Then User |
+|                                  | Transcription               | EnableTranscripting                | DWORD  | Computer, Then User |
+|                                  | Transcription               | EnableInvocationHeader             | DWORD  | Computer, Then User |
+|                                  | Transcription               | OutputDirectory                    | String | Computer, Then User |
+|                                  | UpdatableHelp               | DefaultSourcePath                  | String | Computer Wide       |
+
+I filed [#9632](https://github.com/PowerShell/PowerShell/issues/9632) on UpdatableHelp-DefaultSourcePath to make it allow User settings.
+
+#### JSON file settings format
+
+```json
+{
+  "PowerShell": {
+    "RegularSettings": {
+      "ConsoleSessionConfiguration": {
+        "EnableConsoleSessionConfiguration": true,
+        "ConsoleSessionConfigurationName": "name"
+      },
+      "ProtectedEventLogging": {
+        "EnableProtectedEventLogging": false,
+        "EncryptionCertificate": [
+          "Joe"
+        ]
+      },
+      "ScriptBlockLogging": {
+        "EnableScriptBlockInvocationLogging": true,
+        "EnableScriptBlockLogging": false
+      },
+      "ScriptExecution": {
+        "ExecutionPolicy": "RemoteSigned",
+        "PipelineMaxStackSizeMB": 10
+      },
+      "Transcription": {
+        "EnableTranscripting": true,
+        "EnableInvocationHeader": true,
+        "OutputDirectory": "c:\\tmp"
+      },
+      "UpdatableHelp": {
+        "DefaultSourcePath": "f:\\temp"
+      }
+    },
+
+    "PoliciesSettings": {
+      ...
+    }
+  },
+
+  "OtherPowerShellApplication": {
+    "RegularSettings": {
+      ...
+    },
+    "PolicySettings": {
+      ...
+    }
+}
+```
+
+## Alternate Proposals and Considerations
+
+### Automatically resolve Windows PowerShell policy conflicts
+
+We could attempt to resolve policy conflicts between PowerShell 7 policy and Windows PowerShell policy.
+This would make the `Precedence for Policy settings` not just a simple list but a complex set of rules that would not be easily understood.  See [this conversation](https://github.com/PowerShell/PowerShell/issues/9309?#issuecomment-480643922).
+
+### Allowing environment variable in the JSON
+
+A new RFC should be drafted about how to allow environment variables in the values in the JSON.
+This would allow consistent files across platforms.
+
+### Comment A
+
+Mainly for Unix we'd add `Users` section to computer wide JSON file (`/etc/powershell.config.json`) to allow administrators set policies and regular settings on user level base
+
+```json
+{
+  "PowerShell": {
+    "RegularSettings": {
+      ...
+    },
+    "PolicySettings": {
+      ...
+    },
+  "Users": {
```

Comment:
Would it better to use separate configuration files for each user? Like `smith-powershell.config.json`

Reply:
I think we should have a separate RFC about adding this feature.  cc @iSazonov